isomorphe/static/app.css

```diff
@@ -3,17 +3,50 @@ fieldset {
   border-width: 1px;
 }
 
-.markdown ul + h1,
-.markdown ul + h2,
-.markdown ul + h3,
-.markdown ul + h4,
-.markdown ul + h5,
-.markdown ul + h6 {
-  margin-top: 0.7em;
+.markdown h1,
+.markdown h2,
+.markdown h3,
+.markdown h4,
+.markdown h5,
+.markdown h6 {
+  margin-top: 1em;
 }
 
+
+.markdown p {
+  margin: 0;
+}
+.markdown ul,
+.markdown ol {
+  margin-top: 0.3em;
+  margin-bottom: 0.3em;
+}
+.markdown p + p,
+.markdown ul + p,
+.markdown ol + p {
+  margin-top: 0.6em;
+}
+
+.markdown li {
+  margin-top: 0em;
+  margin-bottom: 0em;
+}
```

Comment:
That is way too tight for my tastes. I agree the default style can feel a bit "airy", but 1. it's the default style and 2. let's not make it too tight as a reaction.

Reply:
Simple one (default Chrome style):

```suggestion
.markdown p {
  margin-block-start: 1em;
  margin-block-end: 1em;
}
```

<img width="1098" alt="Capture d’écran 2024-12-18 à 16 36 51" src="https://github.com/user-attachments/assets/da62b083-500f-45ec-b01d-bd3bd8c49e46" />